Description: Relax rust-ini dependency
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2022-12-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: zram-generator/Cargo.toml
===================================================================
--- zram-generator.orig/Cargo.toml
+++ zram-generator/Cargo.toml
@@ -42,7 +42,7 @@ version = "0.4"
 features = ["std"]
 
 [dependencies.rust-ini]
-version = ">=0.15, <0.18"
+version = ">=0.15"
 [dev-dependencies.ctor]
 version = "0.1"
 
